src/reporting/lambda/custom-resource/quicksight/index.ts

```diff
@@ -514,6 +541,10 @@ const createDataSet = async (quickSight: QuickSight, commonParams: ResourceCommo
       };
     }
 
+    const datasetParameters = buildDataSetParameter(props.dateTimeDatasetParameter);
+
+    logger.info(`datasetParameters: ${JSON.stringify(datasetParameters)}`);
```

Comment:
use `logger.info('datasetParameters: ', {datasetParameters});`

Reply:
This approach is sometimes not very user-friendly; it prints out 'object' rather than displaying detailed content.

Reply:
fixed